gradio/inputs.py

```diff
@@ -290,6 +295,16 @@ def process_example(self, example):
         else:
             return example
 
+    def rebuild_flagged(self, dir, msg):
```

Comment:
`msg` is not used?

Reply:
`msg` here is what I named the input data I get from the api. Changed the name to `data`. This isn't the old message feature